app/code/Magento/Backend/etc/config.xml

```diff
@@ -47,6 +43,13 @@
                 <use_rewrites>0</use_rewrites>
             </seo>
         </web>
+        <dev>
+            <image>
+                <max_quality>4096</max_quality>
```

Comment:
You can't set any value as the max quality. The higher quality that you can use is 100, so 4096 - isn't a very good default value.